java/src/jmri/jmrix/srcp/SRCPSensorManager.java

```diff
@@ -12,23 +13,35 @@
  */
 public class SRCPSensorManager extends jmri.managers.AbstractSensorManager {
 
-    int _bus;
+    //private int _bus = 0; // not used, if needed, get bus via {@link SRCPBusConnectionMemo#memo.getBus()}
```

Comment:
If this is no longer required, just remove it, there isn’t a reason to keep a private variable like this around.